test_bucketeer_cli.py

```diff
@@ -0,0 +1,6 @@
+class TestBucketeerCLI:
+    """Tests for Bucketeer CLI."""
+
+    def test_true(self):
+        """Tests that pytest is setup properly."""
+        assert True
```

Comment:
Create a ticket for a non-placeholder test?

Reply:
I dunno, at this point the logic is so simple that it's not clear to me _what_ to test. The only reason I set this up is because it's set up for Festerize. My inclination is that it's a waste of effort at this point to bother with tests, but I may be overlooking something.

Then, I also wonder whether to remove this no-op test, or leave it.

Reply:
(FWIW, I also feel like it's unnecessary for Festerize at this point)

Reply:
I don't feel like it's a waste of time. One could submit a CSV to Fester through Festerize and confirm that the output CSV is what is expected, for instance. I figured you didn't dive in because spinning up a fester to test against was complicated (and probably worth it's own ticket), but one could also test against a live Fester. Once the basic "does it work?" test is done, I'd start with the options to Fester to make sure they all do what they're supposed to. It might also be good to test what happens when bad parameter values are supplied to those options.

Reply:
Another good thing to test would be the versioning between Fester and Festerize. Does it do what it's supposed to?

Reply:
I don't think it's a waste of time. I thought you'd pushed it off because it was hard (besides just checking response codes from sending good and bad data, one could subscribe to the Slack channel and check that the output actually comes out there -- that's a difficultly level about the basic checks though). Like festerize, I think there is value in confirming that the values we pass as arguments do what we expect them to when a request is sent to Bucketeer.

Reply:
ah, I wasn't even thinking of integration testing. I can see the value now (and the difficulty in writing such tests)

IIIF-940